Data/3_0/Skills/act_int.lua

```diff
@@ -3975,7 +3975,7 @@ skills["IceSpear"] = {
 		},
 	},
 	preDamageFunc = function(activeSkill, output)
-		if activeSkill.skillPart == 3 or activeSkill.skillPart == 4 then
+		if activeSkill.skillPart == 3 or 4 then
```

Comment:
This always evaluates to `true` because it checks if the _number_ 4 is not `false` or `nil`. I don't think that's what you meant to do.